tb_plugin/README.md

```diff
@@ -30,21 +30,32 @@ and give optimization recommendations.
 
 ### Quick Start Instructions
 
-* Start tensorboard
+* Prepare profiling data
 
-  Specify your profiling samples folder.
-  Or you can specify kineto/tb_plugin/samples as an example.
+  You can download [kineto/tb_plugin/samples](https://github.com/pytorch/kineto/tree/master/tb_plugin/samples)
+  to your local and specify it as an example.
```

Comment:
You can download the example profiling data from [kineto/tb_plugin/samples](https://github.com/pytorch/kineto/tree/master/tb_plugin/samples) ?